website/docs/docs/building-a-dbt-project/building-models.md

```diff
@@ -17,6 +17,14 @@ If you're new to dbt, we recommend that you check out our [Getting Started guide
 
 :::
 
+<VersionBlock firstVersion="1.3">
+
+In v1.3, dbt Core is adding support for **Python models**.
```

Comment:
There is no indication in the current docs whether it is possible to use Python models only in Core or in Cloud as well. Is it intentional? If not we could mention to people what version of dbt they need to pick in dbt Cloud.

Reply:
I'll add a note about this to the top of the "Python models" page